recipes/onnx2torch/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "onnx2torch" %}
+{% set version = "1.5.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f668f6ecb2f34029d4c85a66be2f4a329a3d5d2ed8513395e508322d20dcb859
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
+
+requirements:
+  host:
+    - numpy >=1.16.4
+    - onnx >=1.9.0
+    - pip
+    - python
+    - torchvision >=0.9.0
+  run:
+    - numpy >=1.16.4
+    - onnx >=1.9.0
+    - python
+    - torchvision >=0.9.0
+
+test:
+  imports:
+    - onnx2torch
+    - onnx2torch.node_converters
+    - onnx2torch.utils
+
+about:
+  home: "https://github.com/ENOT-AutoDL/onnx2torch"
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE
+  summary: "Nice Onnx to Pytorch converter"
+  doc_url: https://github.com/ENOT-AutoDL/onnx2torch
+  dev_url: https://github.com/ENOT-AutoDL/onnx2torch
+
+extra:
+  recipe-maintainers:
+    - ENOT-AutoDL
```

Comment:
The submitter of the recipe should be the maintainer.

Reply:
:heavy_check_mark: